.github/workflows/push.yml

```diff
@@ -63,6 +63,8 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Pull DEV image
         run: docker pull keboolabot/keboola-processor-split-dev:latest
+      - name: Download modules to the cache
+        run: docker-compose run --rm -u "$UID:$GID" dev go mod download
```

Comment:
V CI bol nejaky problem, ze sa v E2E testoch paralelne kompiloval processor a CLI (to zostava) a paralelne sa stahovali `go modules`.

Ako fix sa stiahnu modules ako prve, a paralelna kompilacia potom uz nie je problem.